configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
Let me open a new PR with one additional change that would let it build on the CI.